app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/step/survey/TerminationSurveyDestination.kt

```diff
@@ -102,174 +133,332 @@ private fun TerminationSurveyScreen(
-                    color = MaterialTheme.colorScheme.onSurfaceVariant,
-                  )
+                  Column {
+                    Row(
+                      verticalAlignment = Alignment.Top,
+                      modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(start = 16.dp, top = 10.dp, end = 16.dp),
+                    ) {
+                      Text(
+                        text = feedback ?: stringResource(id = R.string.TERMINATION_SURVEY_FEEDBACK_HINT),
+                        style = MaterialTheme.typography.bodyLarge,
+                        color = if (feedback != null) {
+                          MaterialTheme.typography.bodyLarge.color
+                        } else {
+                          MaterialTheme.colorScheme.onSurfaceVariant
+                        },
+                        modifier = Modifier.weight(1f),
+                      )
+                    }
+                    Spacer(modifier = Modifier.weight(1f))
+                    Row(
+                      horizontalArrangement = Arrangement.End,
+                      modifier = Modifier
+                        .fillMaxSize()
+                        .padding(start = 16.dp, end = 16.dp, bottom = 12.dp),
+                    ) {
+                      val length = feedback?.length ?: 0
+                      Text(
+                        text = "$length/140",
+                        style = MaterialTheme.typography.titleSmall,
+                        color = MaterialTheme.colorScheme.onSurfaceVariant,
+                      )
+                    }
+                  }
                 }
+                Spacer(modifier = (Modifier.height(4.dp)))
               }
             }
-            Spacer(modifier = (Modifier.height(4.dp)))
           }
         }
       }
+      Spacer(Modifier.height(12.dp))
+      HedvigContainedButton(
+        stringResource(id = R.string.general_continue_button),
+        enabled = uiState.continueAllowed,
+        modifier = Modifier.padding(horizontal = 16.dp),
+        colors = ButtonDefaults.buttonColors(
+          disabledContainerColor = MaterialTheme.colorScheme.surface,
+          disabledContentColor = MaterialTheme.colorScheme.onSurfaceVariant,
+        ),
+        onClick = onContinueClick,
+        isLoading = uiState.isNavigationStepLoading,
+      )
+      Spacer(Modifier.height(16.dp))
+    }
+
+    AnimatedVisibility(
+      visible = uiState.showFullScreenEditText != null,
+      enter = fadeIn() + slideInVertically(initialOffsetY = { it / 2 }),
+      exit = fadeOut(),
+    ) {
+      val reason = uiState.showFullScreenEditText
+      if (reason != null) {
+        FullScreenEditableText(
+          reason.feedBack,
+          modifier = Modifier
+            .fillMaxSize()
+            .windowInsetsPadding(
+              WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal + WindowInsetsSides.Vertical),
+            ),
```

Comment:
So I made it into always dark mode, and now dark mode looks 100% like in figma, and light mode has 2 differences: status bar icons invisible and the text field itself is dark surface color, not light. But from UI point of view, I think it's better like this then with status bar with a different color just hanging there all white. I wonder if this could be done differently, closer to figma. 
![Screenshot_20240514_114656_Hedvig Dev](https://github.com/HedvigInsurance/android/assets/90948269/3c251c51-efba-4912-b4f3-d1e15f7d65d2)
![Screenshot_20240514_114642_Hedvig Dev](https://github.com/HedvigInsurance/android/assets/90948269/be37ed4e-5419-44f8-997f-9dd70cf1b726)